modular_skyrat/modules/exp_corps/code/expeditionary_trooper.dm

```diff
@@ -0,0 +1,121 @@
+	Do not valid hunt using your equipment. Use common sense. Failure to follow these simple rules will result in a job ban.")
+
+/datum/outfit/job/expeditionary_trooper
+	name = "Expeditionary Trooper"
+	jobtype = /datum/job/expeditionary_trooper
+
+	shoes = /obj/item/clothing/shoes/combat/expeditionary_corps
+	ears = /obj/item/radio/headset/headset_sec/alt
+	uniform = /obj/item/clothing/under/rank/expeditionary_corps
+	glasses = /obj/item/clothing/glasses/sunglasses/big
+
+	backpack = /obj/item/storage/backpack
+	satchel = /obj/item/storage/backpack/satchel
+	duffelbag = /obj/item/storage/backpack/duffelbag
+
+	box = /obj/item/storage/box/survival/expeditionary_corps
+
+	backpack_contents = list(/obj/item/exp_corps_equip)
+
+	id = /obj/item/card/id/advanced/silver/exp_corps
+	id_trim = /datum/id_trim/job/expeditionary_trooper
+
+	belt = /obj/item/pda/expeditionary_corps
+
+/obj/effect/landmark/start/expeditionary_corps
+	name = "Expeditionary Trooper"
+	icon_state = "Security Officer"
+
+/obj/item/pda/expeditionary_corps
+	greyscale_colors = "#891417#000099"
+	name = "Military PDA"
+
+/obj/item/storage/box/survival/expeditionary_corps
+	mask_type = /obj/item/clothing/mask/gas/alt
+
+/obj/item/storage/box/expeditionary_survival
+	name = "expedition survival pack"
+	desc = "A box filled with useful items for your expedition!"
+	icon_state = "survival_pack"
+	icon = 'modular_skyrat/modules/exp_corps/icons/survival_pack.dmi'
+	illustration = null
+
+/obj/item/storage/box/expeditionary_survival/PopulateContents()
+	new /obj/item/crowbar/red(src)
+	new /obj/item/kitchen/knife/combat/survival(src)
+	new /obj/item/storage/box/donkpockets(src)
+	new /obj/item/flashlight/glowstick(src)
+	new /obj/item/tank/internals/emergency_oxygen/double(src)
+	new /obj/item/reagent_containers/food/drinks/waterbottle(src)
+	new /obj/item/reagent_containers/blood/universal(src)
+	new /obj/item/reagent_containers/syringe(src)
+
+/obj/structure/closet/crate/secure/exp_corps
+	name = "expeditionary gear crate"
+	desc = "A secure crate, for Expeditionary Corps only!"
+	icon_state = "expcrate"
+	icon = 'modular_skyrat/modules/exp_corps/icons/exp_crate.dmi'
+	req_access = list(ACCESS_GATEWAY, ACCESS_CENT_GENERAL)
+	max_integrity = 5000
+
+/obj/structure/closet/crate/secure/exp_corps/PopulateContents()
+	new /obj/item/storage/firstaid/tactical(src)
+	new /obj/item/storage/box/expeditionary_survival(src)
+	new /obj/item/clothing/suit/space/hardsuit/expeditionary_corps(src)
+	new /obj/item/radio(src)
+	new /obj/item/melee/tomahawk(src)
+	new /obj/item/clothing/gloves/combat/expeditionary_corps(src)
+	new /obj/item/clothing/head/helmet/expeditionary_corps(src)
+	new /obj/item/clothing/suit/armor/vest/expeditionary_corps(src)
+	new /obj/item/storage/belt/military/expeditionary_corps(src)
+	new /obj/item/storage/backpack/duffelbag/expeditionary_corps(src)
+
+/obj/item/exp_corps_equip
```

Comment:
Why put everything in a beacon, and not just the hardsuit?

Reply:
So people feel like they have to equip things.